test/integration/dnspolicy_controller_test.go

```diff
@@ -9,14 +9,16 @@ import (
 
 	. "github.com/onsi/ginkgo/v2"
 	. "github.com/onsi/gomega"
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
```

Comment:
nit: Should be with the rest of the k8s imports

Reply:
done